### DISCUSSION TITLE
Load JSBSim model full in python

### DISCUSSION THREAD

**Discussion by matheusdarocha** [JSBSim-Team/jsbsim#607] [General]
Good morning

I'd like to know if it is possible to load a model at JSBSim without using an external XML file, like putting all inputs as a string for example.

Did someone already run JSBSim without any external file?

Thanks

**Comment by seanmcleod** [comment]
I have done this in the past for someone in 2016. In their case they wanted to encrypt their aerodynamic data etc. and send off a Windows DLL version of JSBSim with their aerodynamic data to an external partner who would use it with Matlab/Simulink to develop the flight control system etc.

What is your use case?

In this case the relevant XML files were embedded as Windows resources in the DLL and encrypted.

![image](https://user-images.githubusercontent.com/824856/159921730-587db86d-db8c-4618-9920-5a479426f851.png)

Examples of the files included as resources in the DLL.

![image](https://user-images.githubusercontent.com/824856/159922536-9db2989d-695e-4621-9a38-d87e79b7cf82.png)
